src/platform/utilities/date/index.js

```diff
@@ -78,3 +78,49 @@ export function timeFromNow(date, userFromDate = null) {
 export function isValidDateString(dateString) {
   return !isNaN(Date.parse(dateString));
 }
+
+const monthIndices = {
+  JAN: 0,
+  FEB: 1,
+  MAR: 2,
+  APR: 3,
+  MAY: 4,
+  JUN: 5,
+  JUL: 6,
+  AUG: 7,
+  SEP: 8,
+  OCT: 9,
+  NOV: 10,
+  DEC: 11,
+};
+
+const LONG_FORM_MONTHS = [
+  monthIndices.MAR,
+  monthIndices.APR,
+  monthIndices.MAY,
+  monthIndices.JUN,
+  monthIndices.JUL,
+];
+
+export const formatDowntime = dateTime => {
```

Comment:
Might be useful to write a little JSDoc saying what format `dateTime` should be in.

Reply:
Good point. It is in flux at the moment because we don't want to be using the `moment.tz` format, but that's currently our temporary solution. It will be fully a string with offset once we get the PagerDuty integration.